### ISSUE TITLE
Get Filled Rows Count

### ISSUE THREAD

**Issue by rrrepos** [theoephraim/node-google-spreadsheet#441]
I could not find a property of a Sheet that will provide the number of filled Rows. When I tried using the `rowCount` property of a sheet that had a single header row, it logged a value like 993!

From what I have read of the v4 API apparently Google does not provide this. Would you consider this as an enhancement to this library in case there is a demand for such a property.

Currently, I am adding a row and gettings. Thanks

**Comment by LuisDev99** [comment]
If you do:

```js
await this.googleSpreadsheet.loadInfo();
this.sheet = this.googleSpreadsheet.sheetsByIndex[MAIN_SPREADSHEET_INDEX];

const rows = await this.sheet.getRows({
      limit: 0,
      offset: STARTING_ROW_INDEX,
});
```

The variable **rows** will contain an array of rows that are **filled**, so to check the amount of rows your spreadsheet has, you just access the length of the variable **rows**:

```js
console.log(rows.length):
```

**Comment by rrrepos** [comment]
Thanks @LuisDev99 : But would not this cause the loading of all the rows in memory? And if they are in a couple of thousands (as in my use-case), would be be a huge memory spike on the server to process the request.

**Comment by LuisDev99** [comment]
Yes, but unfortunately, that is the only way I got around this problem. As you said at the beginning, there is the ```rowcount``` property but it gives the actual amount of rows of the spreadsheet, not the amount of rows that have values inside the spreadsheet.

 The solution I provided works nicely for me because I'm working in a row-based approach, but definitely it is not the case for everyone because of the things you pointed out so it is good to keep that in mind.
